messaging-client-databus/src/main/java/com/inmobi/databus/readers/CollectorStreamReader.java

```diff
@@ -144,6 +144,36 @@ protected CollectorFile getStreamFile(FileStatus file) {
     };
   }
 
+  public Long getPendingSize(Path readTill) throws IOException {
```

Comment:
this is specific to collector stream reader, and is passing Path. I feel it is correct